pom.xml

```diff
@@ -1,66 +1,95 @@
 <?xml version="1.0"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
-	<modelVersion>4.0.0</modelVersion>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
 
-	<groupId>com.esri.geoevent.parent</groupId>
-	<artifactId>track-idle-detector</artifactId>
-	<version>10.6.0</version>
-	<packaging>pom</packaging>
+  <groupId>com.esri.geoevent.parent</groupId>
+  <artifactId>trackidledetector</artifactId>
+  <version>${revision}</version>
+  <packaging>pom</packaging>
 
-	<name>Esri :: GeoEvent :: TrackIdleDector</name>
-	<url>http://www.esri.com</url>
+  <name>${project.artifactId}-parent</name>
```

Comment:
We should not change the name format convention. All GES names are formatted with the `A :: B` convention.

Reply:
Using the ${artifactId} makes it easier to manage if the name ever does change.  But that doesn't happen often, so I can change it back.